Nexus Mods/dist/presence.js

```diff
@@ -0,0 +1,257 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+  return new (P || (P = Promise))(function (resolve, reject) {
+    function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+    function rejected(value) { try { step(generator['throw'](value)); } catch (e) { reject(e); } }
+    function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
+    step((generator = generator.apply(thisArg, _arguments || [])).next());
+  });
+};
```

Comment:
Please remove this, your file is not compiled of TypeScript

Reply:
On it, taken from other presences that use JS. Did the guidelines for that change?

Reply:
Not specifically, but it's source of issues.